### ISSUE TITLE
Error 500: Not Connected

### ISSUE THREAD

**Issue by menih** [kubevious/kubevious#29]
Just follow the installation instructions. 
Installed using Helm.
Installed version of Kubevious is latest (0.7.2).
Went to the page... got "Error 500: Not Connected"

### Environment Details:
 - Kubernetes Vanilla 1.18
 - Browser chrome
 - Version 0.7.2

**Comment by rubenhak** [comment]
@menih, could you run this command and make sure that pods are running?
```sh
$ kubectl get pods -n kubevious
```

should see something like this:

```sh
NAME                                READY   STATUS    RESTARTS   AGE
kubevious-5d877c99bf-7lv2n          1/1     Running   0          17h
kubevious-mysql-0                   1/1     Running   0          8d
kubevious-parser-5496f7c5f7-trg4c   1/1     Running   0          11d
kubevious-ui-6fbb9b6fc9-jdp82       1/1     Running   0          11d
```

**Comment by menih** [comment]
I actually installed it on default namespace. it shouldn't matter though... 

I suspect I know what the issue is... the mysql pod is not starting because of storage. Even though I did specify storageClass ... 

mysql:
  cpu: 50m
  db: kubevious
  image:
    pullPolicy: IfNotPresent
    repository: mysql
    tag: 8.0.19
  memory: 500Mi
  name: kubevious-mysql
  pass: ""
  port: 3306
  storage: 10Gi
  **storageClass: mystorage**
  svcName: kubevious-mysql-svc
  user: root

kubevious-6bcd8d97d7-qbn7b                 1/1     Running    0          10m
kubevious-mysql-0                          0/1     Pending    0          10m
kubevious-parser-6668f98bbc-rrsjk          1/1     Running    0          10m
kubevious-ui-85684994cc-vpb7z              1/1     Running    0          10m

**Comment by menih** [comment]
I suspect the "mysql.storageClass" does not work. Maybe a bug.

**Comment by kubevious** [comment]
@menih, please check the output of
```sh
kubectl get statefulset kubevious-mysql -n kubevious -o yaml
```

In the bottom should include a section like this:
```yaml
  volumeClaimTemplates:
  - metadata:
      creationTimestamp: null
      name: data
    spec:
      storageClassName: mystorage
      accessModes:
      - ReadWriteOnce
      resources:
        requests:
          storage: 30Gi
      volumeMode: Filesystem
    status:
      phase: Pending
```

Also, please include the output of
```sh
kubectl get pvc data-kubevious-mysql-0 -n kubevious -o yaml
```

**Comment by menih** [comment]
I have since corrected the issue manually. So the output you've requested
would reflect the correct information... Just suggested this may be an
issue you may want to look into in the future.

Thanks for your help and responsiveness.

Very interesting product by the way.

Meni

**Comment by rubenhak** [comment]
@menih, could you explain what was the issue exactly and how did you solve it? It would be beneficial for others that might experience similar case. 

Appreciate the feedback! Unrelated to this issue request, but wanted to share a new capability we added to Kubevious recently: https://github.com/kubevious/kubevious/blob/master/docs/rules-engine.md#rules-engine

**Comment by pedroparraortega** [comment]
I faced this issue today. I'm using the helm chart to deploy it and i realised that there is a bug in the helm chart. If you configure a password for the mysql connection it will fail as the mysql statefulset is not using the configmap...

I will try to create a PR for it.

**Comment by rubenhak** [comment]
hi @pedroparraortega, I think that's a separate issue. This particular issue happened with an empty password. 

Indeed, the password was not handled. Could you open a separate issue for supporting password? Would very much appreciate help on this. To make it right, probably best to safe the password in a "Secret" rather than environment variables. Lets store the value of secret in a file (YAML or JSON). Path to the file written in "MYSQL_COFIG" environment variable.

**Comment by kubevious** [comment]
Resolved in [v0.7](https://github.com/kubevious/kubevious/releases/tag/v0.7).
